### DISCUSSION TITLE
Adding a "css class" field in Questionnaires section

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3703] [Ideas]
I think that could be a good idea to add a "css class" field in the question setting in the Questionnaires section (while being Admin) to best target that field in the css custom file...

**Comment by evilaliv3** [comment]
Thank you for your feedback.

In my experience users have typically the need to customize every type of question based on the type.
Can you make an example where having a class for each specific question would result beneficial?

I understand the general idea but the software is already quite complicated and we try to limit development of features that would be rarely used.

**Comment by maelaborazioni** [comment]
[here ](https://github.com/orgs/globaleaks/discussions/3667#discussioncomment-7274875) you can find an example, even if a bit "extreme".
Banally it would be useful even just to change the color of a text, or the font, in order to give more emphasis to certain questions. 
Providing the ability to upload your own CSS is a great feature but it may happen that you don't have a specific selector to refer to.
